Marlin/src/pins/pins.h

```diff
@@ -626,13 +624,13 @@
 #elif MB(BTT_BTT002_V1_0)
   #include "stm32f4/pins_BTT_BTT002_V1_0.h"     // STM32F4                                env:BIGTREE_BTT002
 #elif MB(LERDGE_K)
-  #include "stm32f4/pins_LERDGE_K.h"            // STM32F4                                env:STM32F4
+  #include "stm32f4/pins_LERDGE_K.h"            // STM32F4                                env:LERDGEK
 #elif MB(LERDGE_S)
-  #include "stm32f4/pins_LERDGE_S.h"            // STM32F4                                env:LERDGE_S
+  #include "stm32f4/pins_LERDGE_S.h"            // STM32F4                                env:LERDGES
 #elif MB(LERDGE_X)
-  #include "stm32f4/pins_LERDGE_X.h"            // STM32F4                                env:LERDGE_X
+  #include "stm32f4/pins_LERDGE_X.h"            // STM32F4                                env:LERDGEX
 #elif MB(VAKE403D)
-  #include "stm32f4/pins_VAKE403D.h"            // STM32F4                                env:STM32F4
+  #include "stm32f4/pins_VAKE403D.h"            // STM32F4
```

Comment:
No env here?

Reply:
Right now there simply isn't an appropriate environment for this board. Based on the comments above it is possible that only 10 of these have ever existed, so that is probably ok. It's probably better to have no environment than an incorrect environment.